src/Manager.ts

```diff
@@ -2,8 +2,9 @@ import type { AxiosResponse } from 'axios';
 
 import { HttpClient } from './http/HttpClient';
 import type { ClientConfiguration } from './interfaces/Config';
-import type { ResponseRoot } from './interfaces/AppleMusic/responseRoot';
+import type { ResponseRoot } from './interfaces/AppleMusic';
 import { AppleMusicError } from './AppleMusicError';
+import { Relationship } from './interfaces/AppleMusic';
```

Comment:
file imported twice, maybe then it makes sense to unite them into one?

Reply:
@seth2810 Fixed